src/WebJobs.Script/Config/FunctionsHostingConfigOptions.cs

```diff
@@ -85,6 +85,22 @@ public bool SwtIssuerEnabled
             }
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether non-critical logs should be disabled in the host.
+        /// </summary>
+        public bool RestrictHostLogs
+        {
+            get
+            {
+                return GetFeatureOrDefault(ScriptConstants.HostingConfigRestrictHostLogs, "1") == "1";
```

Comment:
@fabiocav If I am understanding correctly, your requirements would want this to default to false if not set? 

> Default behavior: logs ON
Hosting config set: logs OFF
Hosting config set, user feature flag opting out: ON

Reply:
That's correct.